src/nat/front_ends/fastapi/routes/workflow.py

```
@@ -0,0 +1,539 @@
+
+        return post_openai_api_compatible
+
+    def _job_status_to_response(job: JobInfo) -> AsyncGenerationStatusResponse:
+        if job_store is None:
+            raise HTTPException(status_code=503, detail="Async generation unavailable (job store not configured).")
+        job_output = job.output
+        if job_output is not None:
+            try:
+                job_output = json.loads(job_output)
+            except json.JSONDecodeError:
+                logger.error("Failed to parse job output as JSON: %s", job_output)
```

Ruff (0.14.13)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)